packages/mask/src/components/InjectedComponents/ProfileCard/index.tsx

```diff
@@ -0,0 +1,255 @@
+        },
+        walletItem: {
+            display: 'flex',
+            alignItems: 'center',
+            fontSize: 18,
+            fontWeight: 700,
+        },
+        menuItem: {
+            display: 'flex',
+            alignItems: 'center',
+            flexGrow: 1,
+            justifyContent: 'space-between',
+        },
+        addressMenu: {
+            maxHeight: 192,
+            width: 248,
+            backgroundColor: theme.palette.maskColor.bottom,
+        },
+        addressItem: {
+            display: 'flex',
+            alignItems: 'center',
+        },
+        link: {
+            cursor: 'pointer',
+            marginTop: 2,
+            zIndex: 1,
+            '&:hover': {
+                textDecoration: 'none',
+            },
+        },
+        settingLink: {
+            cursor: 'pointer',
+            marginTop: 4,
+            zIndex: 1,
+            '&:hover': {
+                textDecoration: 'none',
+            },
+        },
+        linkIcon: {
+            color: theme.palette.maskColor.second,
+            fontSize: '20px',
+            margin: '4px 2px 0 2px',
+        },
+        content: {
+            position: 'relative',
+            flexGrow: 1,
+            backgroundColor: theme.palette.maskColor.bottom,
+            overflow: 'auto',
+        },
+        tabs: {
+            display: 'flex',
+            position: 'relative',
+            paddingTop: 0,
+            marginTop: theme.spacing(2),
+        },
+        addressLabel: {
+            color: theme.palette.maskColor.dark,
+            fontSize: 18,
+            fontWeight: 700,
+        },
+        arrowDropIcon: {
+            color: theme.palette.maskColor.dark,
+        },
+        selectedIcon: {
+            color: theme.palette.maskColor.primary,
+        },
+        linkOutIcon: {
+            color: theme.palette.maskColor.secondaryDark,
+        },
+        mainLinkIcon: {
+            margin: '0px 2px',
+            color: theme.palette.maskColor.secondaryDark,
+        },
+        secondLinkIcon: {
+            margin: '4px 2px 0 2px',
+            color: theme.palette.maskColor.secondaryDark,
+        },
+        footer: {
+            position: 'absolute',
+            height: 48,
+            left: 0,
+            bottom: 0,
+            right: 0,
+            display: 'flex',
+            justifyContent: 'flex-end',
+            alignItems: 'center',
+            background: isDark ? 'rgba(0, 0, 0, 0.8)' : 'rgba(255, 255, 255, 0.8)',
+            backdropFilter: 'blur(10px)',
+            padding: theme.spacing(1.5),
+            boxSizing: 'border-box',
+            fontSize: 14,
+            fontWeight: 700,
+            zIndex: 2,
+        },
+        powerBy: {
+            color: theme.palette.text.primary,
+        },
+    }
+})
+
+export const ProfileCard: FC<Props> = ({ identity, ...rest }) => {
+    const classes = useStylesExtends(useStyles(), { classes: rest.classes })
```

Comment:
Please replace the `useStylesExtends` to `useStyle`. Just like:

`const { classes } = useStyles(undefined, { props: { classes: props.classes } })`